datahub-web-react/src/app/ingest/source/builder/RecipeForm/hive.ts

```diff
@@ -2,36 +2,45 @@ import { RecipeField, FieldType } from './common';
 
 export const HIVE_HOST_PORT: RecipeField = {
     name: 'host_port',
-    label: 'Host Port',
-    tooltip: 'host URL.',
+    label: 'Host and Port',
+    tooltip:
+        "The host and port where Hive is running. For example, 'hive:9083'. Note: this host must be accessible on the network where DataHub is running (or allowed via an IP Allow List, AWS PrivateLink, etc).",
     type: FieldType.TEXT,
     fieldPath: 'source.config.host_port',
+    placeholder: 'hive:9083',
+    required: true,
     rules: null,
 };
 
 export const HIVE_DATABASE: RecipeField = {
     name: 'database',
     label: 'Database',
-    tooltip: 'Database (catalog). Optional, if not specified, ingests from all databases.',
+    tooltip: 'Ingest metadata a specific Database.',
     type: FieldType.TEXT,
     fieldPath: 'source.config.database',
+    placeholder: 'my_db',
+    required: true,
```

Comment:
This is not a required field for hive ingestion source. We extract metadata from all databases from hive server if this is not set.

Reply:
Thank you!